### PR TITLE
feat: add security question

### PR DESCRIPTION
**Pull request by frankiefab100**
## Summary of changes

New quiz for Computer science

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

### CODE REVIEW

**Review by jdwilkin4 — COMMENTED** [review]
It looks like you are failing a few of the automated tests.
At quick glance, I think you are failing because you are using the wrong type of quotes for the question. You should use straight quotes like these `""`

But also, you question would be better suited in the `security-quiz` file and not the `cs-quiz` file

**Comment by frankiefab100** [comment]
Okay, I will fix it

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1908,6 +1908,15 @@ const generalCSQuiz = [
...
     Link: "https://www.freecodecamp.org/news/a-guide-to-object-oriented-programming-principles/"
+  },
+  {
+    Question: "Which one is NOT a type of computer virus?",
+    Answer: "Micro virus",
+    Distractor1: "Ransomware",
+    Distractor2: "Worm",
+    Distractor3: "Trojan horse",
+    Explanation: "A computer virus is a type of malware that, when executed, replicates itself by modifying other computer programs and inserting its own code into those programs.",
+    Link: "https://en.wikipedia.org/wiki/Computer_virus"
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:1919]: Please move this question to the security quiz instead

Comment (frankiefab100) [reply src/data/general-cs-quiz.ts:1919]: Alright, I will be doing that

**Review by jdwilkin4 — COMMENTED** [review]
It looks like you have a merge conflict.
Can you resolve that?
Otherwise, this PR should be good

**Comment by frankiefab100** [comment]
> It looks like you have a merge conflict.
> Can you resolve that?
> Otherwise, this PR should be good



> It looks like you have a merge conflict.
> Can you resolve that?
> Otherwise, this PR should be good

Yes, I just found out. Let me resolve it

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**